akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -503,12 +503,14 @@ class Http2ServerSpec extends AkkaSpec with WithInPendingUntilFixed with Eventua
 
       "received SETTINGS_MAX_CONCURRENT_STREAMS" in pending
 
-      "received SETTINGS_HEADER_TABLE_SIZE" in {
+      "received SETTINGS_HEADER_TABLE_SIZE" in new TestSetup with RequestResponseProbes {
         // if the sender of the new size wants to shrink its decoding table, the encoding table on
         // our side needs to be shrunk *before* sending the SETTINGS ACK. So a mechanism needs to be
         // found that prevents race-conditions in the encoder between sending out an encoded message
         // which still uses the old table size and sending the SETTINGS ACK.
-        pending
+        sendSETTING(SettingIdentifier.SETTINGS_HEADER_TABLE_SIZE, Math.pow(2, 15).toInt) // 32768, valid value (between 2^14 and 2^14 - 1)
+
+        expectSettingsAck()
```

Comment:
Added todo, though I assumed the hpack side of things is taken care of by the current impl, but yeah have not really checked this.